qokit/qaoa_objective_maxcut.py

```diff
@@ -68,12 +69,13 @@ def get_qaoa_maxcut_objective(
         parameterized_circuit = get_parameterized_qaoa_circuit(G, p)
     else:
         parameterized_circuit = None
-
+    # Reverse the sign for computing maximizing problem
```

Comment:
Language a bit ambiguous. Let me suggest the following instead: "Reverse the sign as `get_qaoa_objective` assumes that the problem is minimization"